Makefile

```diff
@@ -2,52 +2,30 @@
 
 default: build
 
-charts/generic-prometheus-alerts/ci/rules.yaml: charts/generic-prometheus-alerts/ci/compiled-yaml.yaml
-	@echo "Extracting Rules ..."
-	@cd charts/generic-prometheus-alerts/ci && yq eval 'select(.metadata.name == "test-application-rules") | .spec' compiled-yaml.yaml > rules.yaml
+ci_dir="charts/generic-prometheus-alerts/ci"
+objects=rules app ingress rds sns sqs
+apps=test-application test-business-hours
```

Comment:
we need two apps in order to test properly - one where alerts are only enabled during business hours and the other when they're enabled all of the time